pygeoapi/provider/csv_.py

```diff
@@ -110,15 +110,20 @@ def _load(self, offset=0, limit=10, resulttype='results',
                 return feature_collection
             LOGGER.debug('Slicing CSV rows')
             for row in itertools.islice(data_, offset, offset+limit):
+                try:
+                    coordinates = [
+                        float(row.pop(self.geometry_x)),
+                        float(row.pop(self.geometry_y)),
+                    ]
+                except ValueError:
```

Comment:
Suggest:

```python
msg = f'Skipping row with invalid geometry: {row.get(self.id_field)}'  # noqa
LOGGER.error(msg)
continue
```

Reply:
Included in commit [c855c7c](https://github.com/geopython/pygeoapi/pull/967/commits/c855c7c4eebdf1737bf783f4b535de7a930e40a4)